index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Robert's Tic Tac Toe</title>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<h1>Robert's Tic Tac Toe</h1>
+		<div class="grid">
+		<table>
+			<tr>
+				<td id="1" onclick="makeMove(1)"></td>
```

Comment:
Modern best practices are not to use `onclick` handlers in HTML to setup events. Better to use `addEventListener` in your JS. This is to maintain separation of concerns. HTML should only concern structure and content, not functionality.